Update quickstart.md (#4442)
projectcalico/calico: windows-calico/quickstart.md

  <%

1. Ensure that BGP is disabled.

>    If you installed using tigera-operator, you can do this by:

Reviewer: `If you installed Calico using operator`?

Author: ```suggestion
   If you installed Calico using operator, you can do this by:
```

   ```bash
   kubectl patch installation default --type=merge -p '{"spec": {"calicoNetwork": {"bgp": "Disabled"}}}'
   ```

>    If you installed using the manifest from https://docs.projectcalico.org/manifests/calico-vxlan.yaml then BGP will already be disabled.

Reviewer: How about `BGP is already disabled`?

Reviewer: `If you installed Calico`

Author: ```suggestion
   If you installed Calico using the manifest from https://docs.projectcalico.org/manifests/calico-vxlan.yaml then BGP is already disabled.
```
